app/src/main/java/pro/itshark/moneysplitter/presentation/userProfile/UserModel.kt

```diff
@@ -2,23 +2,28 @@ package pro.itshark.moneysplitter.presentation.userProfile
 
 import android.databinding.BaseObservable
 import android.databinding.Bindable
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
 import pro.itshark.moneysplitter.BR
 import pro.itshark.moneysplitter.EMPTY_STRING
+import pro.itshark.moneysplitter.ZERO_FLOAT
+import pro.itshark.moneysplitter.ZERO_LONG
 import pro.itshark.moneysplitter.model.pojo.UserEntry
 
+@Parcelize
 class UserModel(
-        var id: Long = 0,
-        email: String = EMPTY_STRING,
-        name: String = EMPTY_STRING,
-        surname: String = EMPTY_STRING,
-        phoneNumber: Long = 0,
-        creditCardNumber: Long = 0,
-        balance: Float = 0F,
-        token: String = EMPTY_STRING,
-        imageUrl: String = EMPTY_STRING,
-        backgroundImageUrl: String = EMPTY_STRING
-) : BaseObservable() {
-
+        private val _id: Long = ZERO_LONG,
```

Comment:
Why do u use `_` ?

Reply:
With _ we show that it shod be private, case JS 5 don't have private modifier, oh wait..

Reply:
I use Parcelable and parameters in constructor should be 'var' or 'val', but in this case we can't use custom getters and setters. So I use '_' to fix this problem.